plugins/deployed-labeler/main.go

```diff
@@ -90,36 +92,91 @@ func main() {
 }
 
 func (s *server) markDeployedPR(w http.ResponseWriter, req *http.Request) {
+	var commitSHA string
+	var team string
+
+	for k, v := range req.URL.Query() {
+		switch k {
+		case "commit":
+			commitSHA = v[0]
+		case "team":
+			team = v[0]
+
+		default:
+			s.log.Warnf("Unrecognized parameter received: %s", k)
+		}
+	}
+
+	if team == "" || commitSHA == "" {
+
+		var preconditionFailed struct {
+			Errors []string
+		}
+
+		preconditionFailed.Errors = make([]string, 0, 1)
+		preconditionFailed.Errors = append(preconditionFailed.Errors, fmt.Sprintf("team and commit are required parameters. Team: '%v', commit: '%v'", team, commitSHA))
```

Comment:
What was your goal in creating a new type to handle errors?

Would it make sense to use go standard libraries? Example: the package [errors](https://pkg.go.dev/errors)